### PR TITLE
Add EA forum specific comment colors for dialogues

### PR DESCRIPTION
**Pull request by oetherington**
Screenshot 2023-11-22 at 18 46 38
Screenshot 2023-11-22 at 18 46 48

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1206012727590568) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by agnesstenlund** [comment]
Looks nice!

I'll add a few more colors, realising now we don'y recycle them but just don't show anything 😅 
Screenshot 2023-11-22 at 18 01 01

**Comment by agnesstenlund** [comment]
unrelated, but I also think it looks nice to change the border to 2px so it matches the left line 💡 
Screenshot 2023-11-22 at 18 03 37

**Comment by agnesstenlund** [comment]
damn these are still really hard to tell apart, I'll try some more 😢 
Screenshot 2023-11-22 at 18 05 19

File: packages/lesswrong/themes/siteThemes/eaTheme.ts [hunk 1]
```diff
@@ -54,6 +54,11 @@ export const eaForumTheme: SiteThemeSpecification = {
     text: {
       primaryAlert: "#137283",
       contentHeader: shadePalette.grey[1000],
+      debateComment: "#92C4FF",
+      debateComment2: "#83DFA2",
```
Comment (agnesstenlund) [inline packages/lesswrong/themes/siteThemes/eaTheme.ts:58]:
```suggestion
      debateComment2: "#4BF283",
```

**Comment by agnesstenlund** [comment]
Changed the colors + changed the order, so blue and green aren't the most common combinations 💡

**Comment by agnesstenlund** [comment]
blue and red first now :)

Screenshot 2023-11-23 at 16 23 54

File: packages/lesswrong/themes/globalStyles/globalStyles.ts [hunk 2]
```diff
@@ -188,6 +188,29 @@ const commentsStyle = (theme: ThemeType): JssStyles => ({
   },
 });
 
+
+type Enumerate<N extends number, Acc extends number[] = []> =
+  Acc["length"] extends N
+    ? Acc[number]
+    : Enumerate<N, [...Acc, Acc["length"]]>;
+
+type IntRange<F extends number, T extends number> =
```
Comment (Will-Howard) [inline packages/lesswrong/themes/globalStyles/globalStyles.ts:197]: Looks like this doesn't include the end number
Screenshot 2023-11-24 at 11 15 22

**Review by Will-Howard — APPROVED** [review]
🦭

**Pull request merged**